frontend/src/pages/ErrorsV2/SourcemapErrorDetails/SourcemapErrorDetails.tsx

```diff
@@ -263,7 +264,13 @@ const StackSectionError: React.FC<
 			cursor="default"
 			overflow="scroll"
 			boxShadow="medium"
-			style={{ maxWidth: '450px' }}
+			paddingTop="8"
+			style={{
+				maxWidth: '450px',
+				overflow: 'auto',
+				msOverflowStyle: 'none',
```

Comment:
It's a windows thing.

MacOS doesn't show unnecessary scrollbars anymore, but Windows does. They look a little gross.

![image](https://user-images.githubusercontent.com/878947/234032815-bae55376-2659-4b77-bf71-f452a5c2efe1.png)